### PR TITLE
feat: added support for logits in some classification metrics #215

### PR DESCRIPTION
**Pull request by afoncubierta**
Addresses issue #215

### CODE REVIEW

File: fuse/eval/metrics/libs/classification.py [hunk 1]
```diff
@@ -138,14 +140,20 @@ def accuracy(
     ):
         """
         Compute accuracy score
-        :param pred: class prediction. Each element is an integer in range [0 - num_classes)
-        :param target: the target class. Each element is an integer in range [0 - num_classes)
+        :param pred: class prediction. Each element is an integer in range [0 - num_classes) or a float in range [0-1]
```
Comment (mosheraboh) [inline fuse/eval/metrics/libs/classification.py:143]: It's not a float but a numpy array (dtype=float) - score per class, right?
Can you please also mention that in such a case argmax will be applied?

File: fuse/eval/metrics/libs/classification.py [hunk 2]
```diff
@@ -138,14 +140,20 @@ def accuracy(
...
+        :param target: the target class. Each element is an integer in range [0 - num_classes) or its one hot encoded version
         :param sample_weight: Optional - weight per sample for a weighted score. Each element is  float in range [0-1]
         :return: accuracy score
         """
 
         pred = np.array(pred)
         target = np.array(target)
+
+        if type_of_target(pred) in ("continuous", "continuous-multioutput"):
```
Comment (mosheraboh) [inline fuse/eval/metrics/libs/classification.py:152]: Great

**Review by mosheraboh — CHANGES_REQUESTED** [review]
Thanks! Looks good.
Minor comment about the documentation inline.

**Review by mosheraboh — APPROVED** [review]
Thanks!

**Review by mosheraboh — APPROVED** [review]
